docs/notebook-configuration.md

```diff
@@ -2,16 +2,61 @@
 title: Add metadata to notebooks
 ---
 
-In Jupyter Notebooks you can add cell level configuration by specifying **tags** in the cell metadata.
-There are also global controls in the [project settings](#project-settings).
+In addition to global controls that you can set in the [project
+settings](#project-settings), you can also add metadata to individual notebooks,
+or on notebook cells, to control how MyST handles them during execution and
+rendering.
```

Comment:
I
```suggestion
In addition to global controls that you can set in the [project settings](#project-settings), you can also add metadata to individual notebooks, or on notebook cells, to control how MyST handles them during execution and rendering.
```

Reply:
As a general rule, we try to keep sentences on the same line rather than broken up because of line characters. This helps with linting/formatting and also makes it easier to rearrange lines if needed.